api/android/build-android-lib.sh

```diff
@@ -70,26 +76,27 @@ echo "GStreamer binaries: $GSTREAMER_ROOT_ANDROID"
 echo "NNStreamer root directory: $NNSTREAMER_ROOT"
 
 echo "Start to build NNStreamer library for Android."
+pushd $NNSTREAMER_ROOT
 
 # Modify header for Android
-cd $NNSTREAMER_ROOT
 if ! patch -R --dry-run -sfp1 -i $NNSTREAMER_ROOT/packaging/non_tizen_build.patch; then
-  patch -sfp1 -i $NNSTREAMER_ROOT/packaging/non_tizen_build.patch
+    patch -sfp1 -i $NNSTREAMER_ROOT/packaging/non_tizen_build.patch
 fi
 
 # Make directory to build NNStreamer library
 mkdir -p build_android_lib
 
 # Copy the files (native and java to build Android library) to build directory
-cp -r $NNSTREAMER_ROOT/api/android/* ./build_android_lib
+cp -r ./api/android/* ./build_android_lib
 
 # Get the prebuilt libraries and build-script
 svn --force export https://github.com/nnsuite/nnstreamer-android-resource/trunk/android_api ./build_android_lib
 
-pushd ./build_android_lib
+cd ./build_android_lib
```

Comment:
Let's use ```pushd``` for maintainability. It's especially good if it's nested as we do here.

Reply:
roll back this change